regression/cbmc/Makefile

```diff
@@ -11,6 +13,8 @@ endif
 
 test:
 	@../test.pl -e -p -c "../../../src/cbmc/cbmc --validate-goto-model --validate-ssa-equation" -X smt-backend $(GCC_ONLY)
+	@$(MAKE) -f $(THIS_FILE) test-paths-lifo
+	@$(MAKE) -f $(THIS_FILE) test-cprover-smt2
```

Comment:
How about making the `test` target depend on `test-paths-lifo` and `test-cprover-smt2`?

Reply:
Yes, either that, or make `default` target depend on `test-paths-lifo` and `test-cprover-smt2`?

Reply:
One issue to consider here is: if we add these additional targets as dependencies for `test`, we wouldn't be able to run "just `test`" ...

Reply:
Good call. The right approach probably is to rename the existing `test` target and introduce a new (`PHONY`) `test` target.